internal/util/replaceInCatalinaProps.go

```diff
@@ -0,0 +1,27 @@
+package util
+
+import (
+	"os"
+	"strings"
+)
+
+func ReplaceInCatalinaProps(fileName string) error {
+	input, err := os.ReadFile(fileName)
+	if err != nil {
+		return err
+	}
+
+	lines := strings.Split(string(input), "\n")
+
+	for i, line := range lines {
+		if strings.Contains(line, "common.loader=") {
+			lines[i] = strings.Replace(lines[i], "common.loader=", "common.loader=${BPI_TOMCAT_ADDITIONAL_COMMON_JARS},", 1)
+		}
+	}
```

Comment:
The trouble is that this *could* be a multi-line field. https://stackoverflow.com/a/8978515/1585136

We should collect multi-line fields for input, as a user could supply a custom catalina.properties file. I don't really care how it writes it back out.

Reply:
Ah ha, ok. I'm fine with that. We should probably double quote it. If that's not happening. I didn't check the code, just your example above.

```
common.loader="${BPI_TOMCAT_ADDITIONAL_COMMON_JARS}","${catalina.base}/lib",\
"${catalina.base}/lib/*.jar",\
"${catalina.home}/lib",\
"${catalina.home}/lib/*.jar"
```